geemusic/controllers.py

```diff
@@ -17,5 +18,5 @@ def redirect_to_stream(song_id):
         )
 
     app.logger.debug('URL is %s' % stream_url)
-
-    return redirect(stream_url, code=302)
+    req = requests.get(stream_url, stream=False)
+    return Response(stream_with_context(req.iter_content(chunk_size=1024 * 1024)), content_type=req.headers['content-type'])
```

Comment:
not quite sure what's going on here. are could you maybe add a comment as to why you're handling streaming this way vs using a redirect?